modules/eco-setting-node-maintenance-cr-web-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-maintenance-web-console_{context}"]
+= Setting a node to maintenance mode by using the web console
+
+To set a node to maintenance mode by using the web console, create a `NodeMaintenance` custom resource (CR).
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+* Install the Node Maintenance Operator from the *OperatorHub*.
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Operators* → *Installed Operators*.
+
+. Select the Node Maintenance Operator from the list of operators.
+
+. In the *Node Maintenance* tab, click *Create NodeMaintenance*.
+
+. In the *Create NodeMaintenance* page, select to configure the `NodeMaintenance` CR via *Form view* or *YAML view*.
```

Comment:
Maybe add that the user needs to modify/update the CR for applying the CR on a real node :)

In the *Create NodeMaintenance* page, select to configure the `NodeMaintenance` CR for setting the node's name  via *Form view* or *YAML view*.